.circleci/config.yml.in

```
@@ -223,7 +223,7 @@ binary_common: &binary_common
     wheel_docker_image:
       description: "Wheel only: what docker image to use"
       type: string
-      default: "pytorch/manylinux-cuda102"
+      default: "pytorch/manylinux-cuda116"
```

Shouldn't default be CPU? Or empty, as individual workflows should always specify one they want to build/test against?
```suggestion
      default: ""
```